File: src/main/scala/com/wixpress/embed/mysql/config/RuntimeConfigBuilder.scala
package com.wixpress.embed.mysql.config

import de.flapdoodle.embed.process.config.io.ProcessOutput
import de.flapdoodle.embed.process.io.Processors
import de.flapdoodle.embed.process.runtime.ICommandLinePostProcessor

/**
 * @author viliusl
 * @since 18/09/14
 */
class RuntimeConfigBuilder extends de.flapdoodle.embed.process.config.RuntimeConfigBuilder {

  def defaults(): RuntimeConfigBuilder = {
    processOutput().setDefault(new ProcessOutput(Processors.console(), Processors.console(), Processors.console()))
    commandLinePostProcessor().setDefault(new ICommandLinePostProcessor.Noop)
    artifactStore().setDefault(new ArtifactStoreBuilder().defaults().build())
    return this
  }
}

File: src/main/java/com/wixpress/embed/mysql/config/RuntimeConfigBuilder.java
package com.wixpress.embed.mysql.config;

import de.flapdoodle.embed.process.config.io.ProcessOutput;
import de.flapdoodle.embed.process.io.Processors;
import de.flapdoodle.embed.process.runtime.ICommandLinePostProcessor;

/**
 * @author viliusl
 * @since 27/09/14
 */
public class RuntimeConfigBuilder extends de.flapdoodle.embed.process.config.RuntimeConfigBuilder {

    public RuntimeConfigBuilder defaults() {
        processOutput().setDefault(new ProcessOutput(Processors.console(), Processors.console(), Processors.console()));
        commandLinePostProcessor().setDefault(new ICommandLinePostProcessor.Noop());
        artifactStore().setDefault(new ArtifactStoreBuilder().defaults().build());
        return this;
    }
}